### ISSUE TITLE
Rubick Telekinesis shard bug - IMMIDATE GAME CRUSH

### ISSUE THREAD

**Issue by OTEC-Stikerov** [ValveSoftware/Dota2-Gameplay#3403]
### Ability name

Telekinesis

### Description

Briefly:
Rubick with shard uses Telekines which causes immediate game crush.

You can try it in lobby. In my draft were: Rubick against Shadow Fiend, Shadow Shaman, Faceless Void, Tusk and Earth Spirit.

After a few tries you get a crush directlz to the desktop for sure.

### Example Match ID (and possibly Timestamp)

6774652945

**Comment by shojalilov** [comment]
6774600306 I lost the game due to the bug please fix the bug and give me my mmr please!

**Comment by Luka4ever** [comment]
To be more specific, the bug occurs when you use the [Telekinesis Land] ability once you have Aghanim's Shard, it doesn't occur if you have Quick Cast enabled for the Ability

**Comment by mobisnothere** [comment]
YES this happened to me as well. IT gets quite frustrating after a while.

**Comment by TheUnholyWrath** [comment]
#3346 https://github.com/ValveSoftware/Dota2-Gameplay/blob/main/.github/HELP/HELP.md#before-submitting-a-report
